### PR TITLE
Добавил 100% тестовое покрытие для test_utils.py

### PR DESCRIPTION
**Pull request by SimonOsipov**
```
(venv) ➜  rozental_as_a_service git:(master) python3 -m pytest tests/
======================================================================= test session starts =======================================================================
platform linux -- Python 3.6.5, pytest-4.2.1, py-1.8.0, pluggy-0.13.0
rootdir: /home/simon/Desktop/projects/rozental_as_a_service, inifile:
plugins: cov-2.5.1
collected 12 items                                                                                                                                                

tests/test_integrational.py ..                                                                                                                              [16%]
tests/test_list_utils.py ...                                                                                                                                [41%]
tests/test_strings_extractors.py .....                                                                                                                      [83%]
tests/test_text_utils.py ..
[100%]
```

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -18,20 +23,44 @@ flake8-print==3.1.0
...
+snowballstemmer==2.0.0
+soupsieve==1.9.4
+tabulate==0.8.3
+typed-ast==1.3.5
+typing==3.6.4
+typing-extensions==3.7.2
+uritemplate==3.0.0
+urllib3==1.25.6
+wcwidth==0.1.7
+zipp==0.6.0
```
Comment (Melevir) [inline requirements.txt]: This changes are not required to write tests. Move them to separate pr from here, so we could merge this.

Comment (SimonOsipov) [reply requirements.txt]: Commit reverted

**Comment by Melevir** [comment]
Thanks
